### PR TITLE
Add Trusted Shuffler server

### PR DESCRIPTION
**Pull request by ipetr0v**
This PR adds Trusted Shuffler to the corresponding server binary.

Ref #2614

### CODE REVIEW

File: experimental/trusted_shuffler/scripts/run [hunk 1]
```diff
@@ -2,15 +2,22 @@
...
+cargo build --manifest-path=experimental/trusted_shuffler/server/Cargo.toml
 cargo build --manifest-path=experimental/trusted_shuffler/backend/Cargo.toml
- "${SCRIPTS_DIR}/run_backend" &
-readonly BACKEND_PID=$!
+cargo build --manifest-path=experimental/trusted_shuffler/client/Cargo.toml
 
-cargo build --manifest-path=experimental/trusted_shuffler/server/Cargo.toml
-"${SCRIPTS_DIR}/run_server" &
+RUST_LOG=info cargo run --manifest-path=experimental/trusted_shuffler/server/Cargo.toml -- \
+  --k=1 --listen-address="[::]:8080" --backend-url="http://localhost:8888" &
```
Comment (tiziano88) [inline experimental/trusted_shuffler/scripts/run:10]: k=1 ?

**Review by tiziano88 — APPROVED** [review]

File: experimental/trusted_shuffler/scripts/run (hunk 1 above)
Comment (ipetr0v) [reply experimental/trusted_shuffler/scripts/run:10]: This PR doesn't include the possibility of sending more than 1 request yet

Comment (ipetr0v) [reply experimental/trusted_shuffler/scripts/run:10]: The next PR will be about sending multiple requests from the test client (and checking the delay as well)

File: experimental/trusted_shuffler/scripts/run [hunk 2]
```diff
@@ -2,15 +2,22 @@
...
+sleep 1
```
Comment (mariaschett) [inline experimental/trusted_shuffler/scripts/run:11]: Why do we sleep?

**Review by mariaschett — APPROVED** [review]

File: experimental/trusted_shuffler/scripts/run (hunk 2 above)
Comment (ipetr0v) [reply experimental/trusted_shuffler/scripts/run:11]: Otherwise the background task may run after the client and thus fail

Comment (mariaschett) [reply experimental/trusted_shuffler/scripts/run:11]: I assumed something like that. :) Maybe we want to add a comment?

Comment (ipetr0v) [reply experimental/trusted_shuffler/scripts/run:11]: It's the ultimate Bash synchronization primitive

**Pull request merged**